src/states/common/approved.rs

```diff
@@ -235,7 +238,7 @@ pub trait Approved: Relocated {
                         transition => return Ok(transition),
                     }
                 }
-                NetworkEvent::AckMessage(sec_info) => self.handle_ack_message_event(sec_info)?,
+                NetworkEvent::AckMessage(info) => self.handle_ack_message_event(info)?,
```

Comment:
maybe use `ack_payload` or just `payload` instead of `info`, to be consistent with the variable name in the function definition?